src/components/Editor/SearchBar.js

```diff
@@ -396,8 +406,12 @@ const SearchBar = React.createClass({
   },
 
   buildPlaceHolder() {
-    if (this.state.functionSearchEnabled) {
-      return L10N.getStr("functionSearch.search.placeholder");
+    const { symbolSearchEnabled, selectedSymbolType } = this.state;
+    if (symbolSearchEnabled) {
+      return L10N.getFormatStr(
+        "symbolSearch.search.placeholder",
+        L10N.getStr(`symbolSearch.search.${selectedSymbolType}Placeholder`)
+      );
```

Comment:
It should be safe-ish, but given it's 3 strings vs 1, I would prefer to have three explicit strings for each case and be 100% safe.